packages/devtools_app/lib/src/logging/logging_screen.dart

```diff
@@ -413,6 +415,20 @@ class _MessageColumn extends ColumnData<LogData>
   String getValue(LogData dataObject) =>
       dataObject.summary ?? dataObject.details;
 
+  @override
+  int compare(LogData a, LogData b) {
+    final String valueA = getValue(a);
+    final String valueB = getValue(b);
+    // Matches frame descriptions (e.g. '#12  11.4ms ')
+    final regex = RegExp(r'#\d+\s+\d+.\d+ms\s*');
+    if (valueA.startsWith(regex) && valueB.startsWith(regex)) {
```

Comment:
How do you want to compare if one value is a number and the other isn't? Would it be better to sort all numbers before or after all non-numbers?

Reply:
I think the default string sorting would be fine in this case, which is covered by the end return `valueA.compareTo(valueB)`

Reply:
I think that won't quite work because the sort won't be a consistent ordering.
Imagine you have
```
#2
#2NOTNUM
#1
#1NOTNUM
#10
#10NOTNUM
```
And see what you get when you sort it.

Reply:
Ah whoops - github didn't refresh before I made my last comment. I see now. Let's put the frame numbers before non numbers.

Reply:
Fixed the sort.